README.md

```diff
@@ -47,15 +60,22 @@ $ cd robocup-firmware
 $ ./util/<SYSTEM>-setup
 ```
 
-3) Build the project for the desired target. The `control` target is the firmware for the MBED. The `fpga` target is for the FPGA to be uploaded to the MBED. The `kicker` target is for the kicker MCU to be uploaded to the MBED. The `base` target is for the base station firmware to be uploaded to the base station MBED.
+This will install conan for you if you have not previously installed it. If you respond no to the setup script see the note under Setting up Conan for RoboJackets Firmware in [Getting Started](doc/GettingStarted.md) before continuing.
+
+3) Build the project for the desired target. The `robot` target is the firmware for the MTrain. The `kicker` target is for the kicker MCU to be uploaded to the MTrain. The `clean` target deletes the build directories for both robot and kicker firmware.
```

Comment:
This doesn't actual upload stuff to mtrain and there's extra stuff you have to do for the kicker. I would either describe those here or make a note that there's more steps involved

Reply:
For kicker and the fpga, you have to run the convert.py file which outputs a `*_bin.h` file for the binarys. You throw those in the `robot/lib/Inc/device-bins/` and reupload to mtrain. `make control-upload` is the mtrain upload iirc.

Reply:
I didnt write this part it was just left over I just changed it to say conan is a thing and changed the names to the new build targets.I will say that robot doesnt upload either this section really doesnt explain how to upload. I explain how to manually upload in the FlashingMtrain.md document but I should probably link that here as well as either make one for kicker (I like the seperated single topic md tutorials if you cant tell) or add it to the flashing mtrain one and rename it FlashingHardware.md

Reply:
That should be fine then. I just wanted to make sure it was very clear that more needs to happen before you can restart the robot and hope the code works.